schemas/2016-05-15/Microsoft.DevTestLab.json

```diff
@@ -114,14 +136,29 @@
             }
           ],
           "description": "The properties of the resource."
-        },
-        "id": {
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties"
+      ],
+      "description": "Microsoft.DevTestLab/labs/artifactsources"
+    },
+    "labs_costs": {

```

Comment:
could you update tests/2016-05-15/Microsoft.DevTestLab.tests.json for your changes?